### ISSUE TITLE
Fallback translations are not included in flat map

### ISSUE THREAD

**Issue by m-mak** [Tienisto/slang#80]
**Describe the bug**
Fallback translations are not included in flat map.

**To Reproduce**
Use two translations:
| `translations.i18n.json` | `translations_de.i18n.json` |
|---|---|
| ```{ "test": "Test" }``` | ```{}``` |

Use **build.yaml**:
```yml
targets:
  $default:
    builders:
      fast_i18n:
        options:
          base_locale: en
          fallback_strategy: base_locale
          flat_map: true
```

Generate files: `flutter pub run fast_i18n`

**Result**
The German flat map does not contain the English fallback translations.
`translations.g.dart`
```dart
extension on _TranslationsEn {
	Map<String, dynamic> _buildFlatMap() {
		return <String, dynamic>{
			'test': 'Test',
                }
        }

extension on _TranslationsDe {
	Map<String, dynamic> _buildFlatMap() {
		return <String, dynamic>{}
        }
```

**Expected behavior**
The German flat map should contain the English fallback since it is missing in the German translations.
`translations.g.dart`
```dart
extension on _TranslationsEn {
	Map<String, dynamic> _buildFlatMap() {
		return <String, dynamic>{
			'test': 'Test',
                }
        }

extension on _TranslationsDe {
	Map<String, dynamic> _buildFlatMap() {
		return <String, dynamic>{
			'test': 'Test',
                }
        }
```

**Comment by Tienisto** [comment]
Thanks, can confirm this behaviour

**Comment by Tienisto** [comment]
Fixed in `fast_i18n v5.12.6` and `slang v2.3.1`.

The solution is a bit different than yours but it should work :)
